### ISSUE TITLE
Generated DDL for CREATE DOMAIN is setting an invalid COLLATE of default

### ISSUE THREAD

**Issue by jimbogithub** [dbeaver/dbeaver#12325]
#### System information: 
- Operating system (distribution) and version: macOS 11.3
- DBeaver version: 21.0.3.202104181339
- Additional extensions: none

#### Connection specification:
- Database name and version: PostgreSQL 12.6
- Driver name: PostgreSQL JDBC Driver 42.2.5
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:

Generated DDL for `CREATE DOMAIN` is setting an invalid `COLLATE` of `default`, e.g:  
```
CREATE DOMAIN "ct_ccy" AS character varying(8)
	COLLATE default;
```
PostgreSQL rejects that.  It appears it needs to be quoted like so:
```
CREATE DOMAIN "ct_ccy" AS character varying(8)
	COLLATE "default";
```
Or given it's the default, just left off entirely:
```
CREATE DOMAIN "ct_ccy" AS character varying(8);
```

**Comment by LonwoLonwo** [comment]
Hello @jimbogithub 
Thanks for the bug report!

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.0.5
Will be available [here](https://dbeaver.io/) at 2021-05-17

**Comment by uslss** [comment]
verified
